libs/deer/tests/test_impls_core_integral.rs

```diff
@@ -114,6 +114,18 @@ proptest_integral!(
     proptest_try_fit!(i8 :: I8Deserializer, i16 :: I16Deserializer, i32 :: I32Deserializer, i64 :: I64Deserializer, i128 :: I128Deserializer, u128 :: U128Deserializer);
 );
 
+// the code for pointer width 16, 32, 128 is pretty much the same, the tests only differ
+// while we know that u32 and u64 will fit, there are no `From<T>` implementations, so we need
+// to fall back to `proptest_try_fit!` which uses `TryFrom<T>`
+// The reason we have no other use-cases for the other `target_pointer_width`s is because it is hard
+// to emulate those during testing.
+#[cfg(target_pointer_width = "64")]
+proptest_integral!(
+    Token::Number(usize as u64);
+    proptest_fit!(u8 :: U8Deserializer, u16 :: U16Deserializer);
+    proptest_try_fit!(i8 :: I8Deserializer, i16 :: I16Deserializer, i32 :: I32Deserializer, u32 :: U32Deserializer, u64 :: U64Deserializer, i64 :: I64Deserializer, i128 :: I128Deserializer, u128 :: U128Deserializer);
```

Comment:
Should `u32` be present for `fit` instead of `try_fit`?
Also why is `u64` itself in the `try_fit`, should that also just fit/be an identity?

Reply:
correct, seems like I didn't correctly modify my copy pasted macro invocation^^ (fixed in [3c193fd](https://github.com/hashintel/hash/pull/2430/commits/3c193fd03236cc70fe7b1e811eec6893e628e64e))